### PR TITLE
NOISSUE - Add VerneMQ support

### PR DESCRIPTION
**Pull request by drasko**
Signed-off-by: drasko <drasko.draskovic@gmail.com>

Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
Please review the [CONTRIBUTING.md](./CONTRIBUTING.md) file for detailed contributing guidelines.

### What does this do?
Adds support for [VerneMQ](https://github.com/vernemq/vernemq), to be used as an alternative MQTT adapter, instead of [Aedes](https://github.com/mcollina/aedes)

### Which issue(s) does this PR fix/relate to?
Put here `Resolves #XXX` to auto-close the issue that your PR fixes (if such)

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
Yes

### CODE REVIEW

File: mqtt/verne/src/mfx_auth.erl [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    %% 4. return {ok, [{ModifierKey, NewVal}...]} -> PUBLISH is authorized, but we might have changed different Publish Options:
+    %%     - {topic, NewTopic::string}
+    %%     - {payload, NewPayload::binary}
+    %%     - {qos, NewQoS::0..2}
+    %%     - {retain, NewRetainFlag::boolean}
+    %% 5. return {error, whatever} -> auth chain is stopped, and message is silently dropped (unless it is a Last Will message)
+    %%
+
+    % Topic is list of binaries, ex: [<<"channels">>, <<"1">>, <<"messages">>, <<"subtopic_1">>, ...]
+    [{cnanel_id, ChannelId}, {content_type, ContentType}, {nats_subject, NatsSubject}] = parseTopic(Topic),
```
Comment (anovakovic01) [inline mqtt/verne/src/mfx_auth.erl]: Fix typo `cnanel_id`. In other places too.

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

**Review by dborovcanin — CHANGES_REQUESTED** [review]
Do we really need `rebar3` (mqtt/verne/rebar3) binary?

File: mqtt/verne/README.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      environment:
+        MF_MQTT_ADAPTER_LOG_LEVEL: ${MF_MQTT_ADAPTER_LOG_LEVEL}
+        MF_MQTT_INSTANCE_ID: mqtt-adapter
+        MF_MQTT_ADAPTER_PORT: ${MF_MQTT_ADAPTER_PORT}
+        MF_MQTT_ADAPTER_WS_PORT: ${MF_MQTT_ADAPTER_WS_PORT}
+        MF_MQTT_ADAPTER_REDIS_URL: tcp://mqtt-redis:${MF_REDIS_TCP_PORT}
+        MF_MQTT_ADAPTER_ES_URL: tcp://es-redis:${MF_REDIS_TCP_PORT}
+        MF_NATS_URL: ${MF_NATS_URL}
+        MF_THINGS_AUTH_HTTP_URL: http://things:${MF_THINGS_AUTH_HTTP_PORT}
+        DOCKER_VERNEMQ_PLUGINS__VMQ_PASSWD: "off"
```
[the remark this answers is not in the document]
Comment (drasko) [reply mqtt/verne/README.md:60]: @dusanb94 this does not come from us, this is VerneMQ specific - a way to tweak params in the `vernemq.config` withing the container. This is their converntion.

**Comment by drasko** [comment]
> Do we really need `rebar3` (mqtt/verne/rebar3) binary?

@dusanb94 we do not, but it is a script, and VerneMQ keep it in thier demo plugin that I started with: https://github.com/vernemq/vernemq_demo_plugin - so I decided that we keep it.

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**